.github/workflows/lychee_url_checker.yml

```diff
@@ -0,0 +1,27 @@
+name: Check URLs with Lychee
+
+on:
+  push:
+  pull_request:
+  schedule:
+    # Run everyday at 3 am UTC
+    - cron: 0 3 * * *
+
+jobs:
+  linkChecker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Lychee URL checker
+      # use stable version for now to avoid breaking changes
+      # sha-256 hash of latest stable version for security reasons
+        uses: lycheeverse/lychee-action@d322e38893906cef8788eefe9c63694c562c7dfa
+        with:
+          # fail the action on broken links
+          fail: true
+        env:
+        # to be used in case rate limits are surpassed
+          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+          # arguments with file types to check
+          args: --no-progress  './**/*.rst' './**/*.md' './**/*.py' './**/*.ipynb' --timeout 10 --max-retries 5
```

Comment:
It is usually a good practice ([1](https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file), [2](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)) to leave an empty line at the end of code files.

Reply:
Noted!